src/thunderbots/software/util/ros_messages.cpp

```diff
@@ -1,5 +1,8 @@
 #include "util/ros_messages.h"
 
+#include "ros_messages.h"
```

Comment:
Is this including the same thing as the `#include` above? In any case, this should use the absolute path not the relative path.

Reply:
fixed the include

Reply:
I think i fixed the thing with the exception too